.changeset/new-buttons-join.md

```diff
@@ -0,0 +1,5 @@
+---
+"@shopware-ag/meteor-component-library": patch
+---
+
+Test coverage for x-small mt-button
```

Comment:
I would re-write this to something that more user-facing like add x-small variant to button component or allow x-small as valid variant for button